guide/english/algorithms/binary-search-trees/index.md

```diff
@@ -4,10 +4,10 @@ title: Binary Search Trees
 ## Binary Search Trees
 ![Binary Search Tree](https://cdn-images-1.medium.com/max/1320/0*x5o1G1UpM1RfLpyx.png)
 
-A tree is a data structure composed of nodes that has the following characteristics:
-1. Each tree has a root node (at the top) having some value.
+A tree is a data structure composed of nodes that have the following characteristics:
```

Comment:
> A tree is a data structure composed of nodes that have the following characteristics:

The original **has** referred to **data structure** not to **nodes**.  Please correct.

> A tree is a data structure composed of nodes that have the following characteristics:

Same issue above.  Revert to the original.